### PR TITLE
Addon-viewports: Add back default viewports

### PR DESCRIPTION
**Pull request by ndelangen**
Issue:
- viewports no longer had any defaults
- there was an issue where if the viewport width was > main area, the viewport wouldn't render at full width.

## What I did
- added a minimal default of 3 viewports as a sane list of viewports, hopefully most people will find useful.
- fixed the layout issue, by using css-grid. flexbox fails for this usecase. This should fail gracefully on browsers that don't support it: they'd just not render the iframe centered.

### CODE REVIEW

**Review by shilman — CHANGES_REQUESTED** [review]
@ndelangen Why does this need to be a breaking change? Let's not remove `withViewport` until 6.0? And what's the problem with just using `INITIAL_VIEWPORTS`?

**Comment by ndelangen** [comment]
withViewport is already not exported... Might have happened in the TS migration?

File: addons/viewport/src/Tool.tsx [hunk 1]
```diff
@@ -195,12 +196,15 @@ export const ViewportTool: FunctionComponent<{}> = React.memo(
                 },
                 [`#${wrapperId}`]: {
                   padding: theme.layoutMargin,
-                  display: 'flex',
                   alignContent: 'center',
                   alignItems: 'center',
                   justifyContent: 'center',
```
Comment (wuweiweiwu) [inline addons/viewport/src/Tool.tsx]: if its not a flex container do we still need justify content and align items?

Comment (ndelangen) [reply addons/viewport/src/Tool.tsx]: `display: grid`

**Pull request merged**